framework/source/class/qx/test/Interface.js

```diff
@@ -365,7 +363,7 @@ qx.Class.define("qx.test.Interface",
             members :
             {
               getValue : function() {},
-              setValue : function(value) {}
```

Comment:
Really? A setValue function that doesn't take a value? If lint is complaining about value being unused, this is the wrong fix. If the function really needn't do anything, then I'd declare it as:
```
setvalue : function(value) { value = value; }
```
(Then we'll see if lint complains about a "useless statement"...)